src/components/JobsPanelParameters/JobsPanelParameters.js

```diff
@@ -25,14 +25,13 @@ const JobsPanelParameters = ({
   const [selectedParameter, setSelectedParameter] = useState({})
 
   const handleAddNewParameter = () => {
-    const emptyParameter = Object.values(newParameter).filter(
-      value => value.length > 0
-    )
+    const emptyParameter =
+      !newParameter.name.length || !newParameter.value.length
 
-    if (emptyParameter.length === 0) {
+    if (emptyParameter) {
       setNewParameter({
         name: '',
-        type: '',
+        type: 'string',
```

Comment:
You should either use `id` here and get label wherever you need. Or use label here. If you use label here you can't hardcode it.